README.md

```diff
@@ -1,38 +1,50 @@
 # Weather Greeting Application
-This project demonstrates how to use Python to create web application. It is a primarily meant for educational purposes, specifically created for the WorldQuant University Introduction to Data Science Module.
+
+This project demonstrates how to use Python to create a web application. It is primarily meant for educational purposes, specifically created for the *WorldQuant University - Introduction to Data Science* Module.
 
 The skills and tools used are:
+
 * Python's `requests` package.
 * Python's `flask` web framework.
-* working with JSONs.
+* working with JSON.
 * modular programming
 
-When a client visits the application, it
+When a client visits the application, it:
+
 1. gets the client's IP address.
-1. uses the IP address to look up their location.
-1. uses their location data to greet them with temperature of the city they are located in.
+2. uses the IP address to look up their location.
+3. uses their location data to greet them with temperature of the city they are located in.
 
 ## Prerequisites
-All required Python packages can be found in the `requirements.txt` file. Additionally, the provided `Makefile` can be used to created a virtual environment by running `make venv`. You will also need a Heroku account and have installed the Heroku CLI. For more information on the Heroku CLI, go to https://devcenter.heroku.com/articles/heroku-cli#download-and-install.
+
+All required Python packages can be found in the `requirements.txt` file. Additionally, the provided `Makefile` can be used to create a virtual environment by running `make venv`. You will also need a Heroku account, and to have installed the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli#download-and-install).
 
 ## Running the app locally using Flask
+
 You may want to run the app using Flask locally before deploying it to Heroku, especially if you have made any changes to the code. To run locally:
 
 1. clone the repository.
 1. in the repository, run `make deploy`.
 1. open the link provided in the command line.
 
 If you are using Windows, you can:
+
 1. create and activate the virtual environment.
-1. `set FLASK_APP=weather_app/app.py` in the command line.
-1. run `python -m flask run`.
-1. open the link in the command line.
+
+        py -3 -m venv venv
+        venv\Scripts\activate.bat
+
+2. `set FLASK_APP=weather_app/app.py` in the command line.
+3. run `python -m flask run`.
+4. open the link in the command line.
```

Comment:
good, just make sure the numbering is all `1.`

Reply:
Okay, I'll restore all the 1's I had replaced.